guides/doc-Content_Management_Guide/topics/Importing_Content.adoc

```diff
@@ -1,28 +1,272 @@
-[[Importing_Red_Hat_Content]]
-== Importing Red{nbsp}Hat Content
+[[Importing_Content]]
+== Importing Content
 
-This section describes how to use products and repositories in {Project}, and to create synchronization plans to ensure your {Project} content remains up to date with the content on the Red{nbsp}Hat Content delivery network (CDN).
+This chapter outlines how you can import {customcontent} of different types to {Project}.
+If you want to import Rpms, Files, or different content types to {Project}, it is done with largely the same procedures in this chapter.
```

Comment:
```suggestion
If you want to import RPMs, files, or different content types to {Project}, the underlying procedures are the same:
```

@jlsherrill when we met about Katello docs, we talked about how what was missing was context/a scenario/ a defined use case. You mentioned how users need to realise that they don't need to always do CVs, that syncing content might be enough for people. Do you think that we could create a list here of all the scenarios that importing content will serve? I can use that content as part of an introduction with a link to this section also ? Or if you prefer we can do this as part of another PR, that is fine, but since we've tackled an entire chapter in this section, it would be nice to contextualise why and who this is important to.

Reply:
Thats a great idea. i'll add some context!